### DISCUSSION TITLE
raspicat_documentation/chat_corner/news/

### DISCUSSION THREAD

**Discussion by giscus** [CIT-Autonomous-Robot-Lab/raspicat_documentation#17] [Announcements]
# raspicat_documentation/chat_corner/news/

https://uhobeike.github.io/raspicat_documentation/chat_corner/news/

**Comment by uhobeike** [comment]
Raspberry Pi Catに搭載されていたRaspberry Piを全てRaspberry Pi 4に取り替えました。

**Reply by uhobeike** [reply]
なのでRaspberry Pi Catだけど、
Raspberry Pi Cat V2の環境構築方法や実行方法を見たほうが良かったりする場合があります。。。

**Comment by TonoLeo** [comment]
https://github.com/CIT-Autonomous-Robot-Lab/raspicat_gamepad_controller
長時間のマッピングに対して操作が厳しいと感じたので
Ｂでの直進、Ｙでの停止、スティックによる角度調節機能を追加しました。

**Comment by uhobeike** [comment]
これから、つくばチャレンジ2023のための大規模な（そうでもないかも）修正を行う

やること
・B3がマニュアルを読んで一からRaspberry Pi Catのセットアップをできるようにする
・B3がつくばチャレンジの会場で困った時に参考になるマニュアルを作成する
